### PR TITLE
Explorer: Easier, more cohesive, aggregator controls

### PR DESCRIPTION
**Pull request by Alenar**
## Content

This PR reworks the explorer upper part (that contains the aggregator selection, status and refresh interval setter) to make it more easier to use and free some space:

- Group refresh interval, aggregator selection, aggregator status/epoch setting into a `ControlPanel` component to make the code more cohesive
- Replace `Aggregator Status` and `Epoch Setting` titles with a `Aggregator` title that is before the aggregator and refresh interval selectors 
- Remove labels on the aggregator selection and refresh interval setter to free some vertical space
- Interval setter:
  - Replace `play/pause` button with a `Refresh` button
  - Add a `Off` value to the available options to stop the automatic refresh
  - Add a tool-tip on the available refresh interval options to compensate the removal of the label

  Previous button was ambiguous and made little sense and if we only want to refresh once we had to click on it twice, by regrouping all the interval values in the options and adding a dedicated button the UI should be easier to use.
- Make the `EpochSettings` component collapsible the same way as the aggregator status _(for aggregators that does not yet support the `/status` route)_

Demo:
[Screencast from 2024-11-19 16-25-00.webm](https://github.com/user-attachments/assets/244652ba-13a1-458f-a9f8-a77e0182dbec)

## Pre-submit checklist

- Branch
  - [x] Tests are provided (if possible)
  - [x] Crates versions are updated (if relevant)
  - [ ] CHANGELOG file is updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested

### CODE REVIEW

**Review by sfauvel — APPROVED** [review]
LGTM

File: mithril-explorer/src/components/ControlPanel/IntervalSetter/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <Button type="button" onClick={() => dispatch(changeRefreshSeed())} variant="primary">
+          Refresh
+        </Button>
+        <OverlayTrigger overlay={<Tooltip>Set auto-refresh interval</Tooltip>}>
+          <Form.Select
+            aria-label="Auto refresh interval"
+            value={updateInterval}
+            onChange={(e) => dispatch(setUpdateInterval(parseInt(e.target.value)))}>
+            <option value={undefined}>Off</option>
+            <option value={1000}>1 seconds</option>
```
Comment (jpraynaud) [inline mithril-explorer/src/components/ControlPanel/IntervalSetter/index.js:21]:
```suggestion
            <option value={1000}>1 second</option>
```

**Review by jpraynaud — APPROVED** [review]
LGTM

**Review by dlachaume — APPROVED** [review]
LGTM 🚀

**Pull request merged**